# Global Planner Configuration Parameters
NavfnROS:
    visualize_potential: false
